api_server/db/migrations/versions/d367b3e690be_scale_of_balance.py

```diff
@@ -0,0 +1,33 @@
+"""scale of balance
+
+Revision ID: d367b3e690be
+Revises: d4111c8366d2
+Create Date: 2022-05-10 07:38:49.530082
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'd367b3e690be'
+down_revision = 'd4111c8366d2'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('Balances', 'amount',
+               existing_type=sa.NUMERIC(precision=2, scale=0),
+               type_=sa.Numeric(precision=4, scale=2),
+               existing_nullable=True)
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('Balances', 'amount',
+               existing_type=sa.Numeric(precision=4, scale=2),
+               type_=sa.NUMERIC(precision=2, scale=0),
+               existing_nullable=True)
```

Comment:
It looks like part of migrations makes minor changes like altering columns etc. Probably we can compress this changes into one - two migrations that makes all this stuff?

Reply:
First task to next sprint will be compress all migrations to one